test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -409,7 +409,7 @@ describe('acceptSession()', () => {
 
     session.emit('terminated');
 
-    expect(mockAudioElement.parentNode.removeChild).not.toHaveBeenCalledWith(mockAudioElement);
+    expect(mockAudioElement.parentNode!.removeChild).not.toHaveBeenCalledWith(mockAudioElement);
```

Comment:
Codacy has a fix for the issue: [Forbidden non-null assertion.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    expect(mockAudioElement.parentNode?.removeChild).not.toHaveBeenCalledWith(mockAudioElement);
```